Dockerfile.shell

```diff
@@ -0,0 +1,21 @@
+FROM python:3.5
+
+RUN groupadd user && useradd --create-home --home-dir /home/user -g user user
```

Comment:
This code is not going to be executed within the host machine, but rather within the docker container, which is always going to be a Linux OS whether it is run on Mac or Linux. (In some ways a container is like a virtual machine and so these commands are setting up the environment within it.)